docs/Clearwater_stress_testing.md

```diff
@@ -12,82 +12,78 @@ package installed.
+## Deploying a stress node
 
-### How they work
+Follow [this process](https://github.com/Metaswitch/crest/blob/dev/docs/Bulk-Provisioning%20Numbers.md) to bulk provision the number of subscribers you want. As a general guideline, we'd expect a small deployment (with one Sprout, Homestead and optionally a Ralf node, each with one full CPU core) to handle at least 50,000 subscribers.
 
-The clearwater-sip-stress package includes two important scripts.
+You should also ensure that the `reg_max_expires` setting is set to 1800 rather than the default of 300 - see our [configuration guide](Clearwater_Configuration_Options_Reference.md) for instructions. This matches the assumptions in our load profile of 2 re-registers per hour.
 
-* `/usr/share/clearwater/infrastructure/scripts/sip-stress`, which generates a `/usr/share/clearwater/sip-stress/users.csv.1` file containing the list of all subscribers we should be targeting - these are calculated from properties in `/etc/clearwater/shared_config`.
-* `/etc/init.d/clearwater-sip-stress`, which runs `/usr/share/clearwater/bin/sip-stress`, which in turn runs SIPp specifying `/usr/share/clearwater/sip-stress/sip-stress.xml` as its test script. This test script simulates a pair of subscribers registering every 5 minutes and then making a call every 30 minutes.
+If you're using chef, create your stress test node by typing `knife box create -E ENVIRONMENT sipp --index 1`.
 
-The stress test logs to `/var/log/clearwater-sip-stress/sipp.<index>.out`.
+Otherwise, follow these steps to deploy a stress node manually:
 
-## Running Stress
+* create a new virtual machine and bootstrap it [by configuring access to the Project Clearwater Debian repository](Manual_Install.md#configure-the-apt-software-sources).
+* set the following properties in /etc/clearwater/local_config:
+    * (required) local_ip - the local IP address of this node
+    * (optional) node_idx - the node index (defaults to 1)
+* run `sudo apt-get install clearwater-sip-stress` to install the Debian package.
 
-### Using Chef
+The stress node needs to be able to open connections to Sprout on TCP ports 5052 and 5054, and Sprout needs to be able to open connections to the stress node on port 5082.
 
-This section describes step-by-step how to run stress using Chef automation.  It includes setting up a new deployment.  It is possible to run stress against an existing deployment but you'll need to reconfigure some things so it's easier just to tear down your existing deployment and start again.
+## Running stress
 
-1.  If you haven't done so already, set up chef/knife (as described in [the install guide](Automated_Install.md)).
-2.  cd to your chef directory.
-3.  Edit your environment (e.g. `environments/ENVIRONMENT.rb`) to override attributes as follows.
+To kick off a stress run, simply run: `/usr/share/clearwater/bin/run_stress <home_domain> <number of subscribers> <number of minutes>`. 
```

Comment:
It's not immediately obvious what the minutes option is. Also, is there an option to let stress run until it's manually stopped? Can there be?